src/server/verification/cron/DisposeEnrollmentsPeriodic.js

```diff
@@ -0,0 +1,80 @@
+import moment from 'moment'
+import AdminWallet from '../../blockchain/AdminWallet'
+import UserDBPrivate from '../../db/mongo/user-privat-provider'
+import getTasksRunner from '../../cron/TaskRunner'
+import Logger from '../../../imports/logger'
+import Config from '../../server.config'
+
+import createEnrollmentProcessor from '../processor/EnrollmentProcessor'
+
+const logger = Logger.child({ from: 'disposeEnrollmentsPeriodic' })
+
+export class DisposeEnrollmentsPeriodicTask {
+  get schedule() {
+    return Config.enrollDisposalCron
+  }
+
+  get name() {
+    return 'DisposeEnrollmentPeriodic'
+  }
+
+  async execute({ setTime }) {
+    try {
+      logger.info('Starting worker', this.name)
+      const authenticationPeriod = await this.getAuthenticationPeriod()
+      logger.info('Authentication period found', authenticationPeriod)
+      const fvRecords = await this.getFVRecords(authenticationPeriod)
+      logger.info(`Fetched ${fvRecords.length} FV records from db`)
+      const disposeEnrollments = await this.disposeEnrollments(fvRecords)
+      logger.info(`Performed Dispose Enrollment for ${disposeEnrollments.length} enrollmentIds`)
+      const enrollmentIdentifiers = disposeEnrollments.reduce((arr, disposeEnrollment) => {
```

Comment:
use .filter

Reply:
using filter will produce an array of objects with enrollmentIdentifier and status, what i wanted is an array of enrollmentIdentifiers to send as an argument to removeRecords method